proto/zitadel/settings.proto

```diff
@@ -67,9 +66,16 @@ message TwilioConfig {
   string sender_number = 2;
 }
 
-
 enum SMSProviderConfigState {
   SMS_PROVIDER_CONFIG_STATE_UNSPECIFIED = 0;
   SMS_PROVIDER_CONFIG_ACTIVE = 1;
   SMS_PROVIDER_CONFIG_INACTIVE = 2;
 }
+
+message OIDCConfig {
```

Comment:
I think having the same object name for different use cases is a bad idea. My best alternative name by now is `OIDCSettings`